### ISSUE TITLE
Version 8.x-2.00 stable

### ISSUE THREAD

**Issue by Ambient-Impact** [pdenooijer/hook_event_dispatcher#130]
Not to rush things, but I'm wondering @pdenooijer if you have criteria for when you consider 8.x-2.x ready to release a stable version? Let me know if there are specific things that need doing that I can help with, be they code or bouncing ideas around.

**Comment by pdenooijer** [comment]
Hmm I am thinking about releasing v2 as is and do the ModuleHandler in v3. What do you think about that?

**Comment by Ambient-Impact** [comment]
I think releasing 8.x-2.0 as is makes sense. Does the ModuleHandler introduce any breaking changes? I'm under the impression that it shouldn't - although I guess the dynamic hook functionality with event factories and all that may depending on how it works. If it's possible to do that without breaking changes (i.e. just new features), it could be released in like 8.x-2.1 or some other minor version, otherwise it would be 8.x-3.0.

tl;dr sounds good to me. 😃

**Comment by pdenooijer** [comment]
I guess it will result in v3, but that is fine as well. Seems like a big upgrade to me 🙂 !

I think the best way to tackle this to start with a beta release and see if that works for a while and then change it to final v2 to make sure it does not have bugs. Any thoughts about that?

**Comment by Ambient-Impact** [comment]
A beta seems like the way to go, given the changed namespaces and other stuff. 🚀

**Comment by pdenooijer** [comment]
[2.00-beta1](https://www.drupal.org/project/hook_event_dispatcher/releases/8.x-2.00-beta1) 🎉 🎊

**Comment by Ambient-Impact** [comment]
Awesome! Will test it out one of these days.

**Comment by Ambient-Impact** [comment]
Initial testing on my own site running 8.x-2.00-beta2 seems to have no issues thus far. 🎉 

One suggestion for the release notes: make it really obvious that most of the events have changed namespaces. I know it's a major version and backwards compatibility out of the box is not expected, but it might help some people understand why they're suddenly getting WSODs and errors - i.e. devs with more intermediate or less advanced skills.

**Comment by pdenooijer** [comment]
Good point, I'll update the release notes.

Done, I hope this makes it more clear.

**Comment by Ambient-Impact** [comment]
Great, thanks!

**Comment by Ambient-Impact** [comment]
Been using it for over a week now without any issues. Hopefully other people have been testing it. @pdenooijer, is there anything else left to do before a stable release?

**Comment by pdenooijer** [comment]
Nope don't think so. Fixed all bug reports so far 🙂. End of the month seems like a good moment for the stable release, agreed?

**Comment by Ambient-Impact** [comment]
I'd personally like it sooner, but that's up to you. 🚀

**Comment by pdenooijer** [comment]
Will release version 2.00 now 🙂

**Comment by Ambient-Impact** [comment]
![Someone waving things in excitement](https://user-images.githubusercontent.com/2289006/82493726-6eb06200-9ab6-11ea-8b30-70ac1eba7188.gif)
